cmd/queue/main.go

```diff
@@ -302,7 +303,7 @@ func main() {
 	}()
 
 	// Open a websocket connection to the autoscaler
-	autoscalerEndpoint := fmt.Sprintf("ws://%s.%s:%d", servingAutoscaler, system.Namespace, servingAutoscalerPort)
+	autoscalerEndpoint := fmt.Sprintf("ws://%s.%s.svc.%s:%d", servingAutoscaler, system.Namespace, utils.GetClusterDomainName(), servingAutoscalerPort)
```

Comment:
Why this change? (or why part of this pr?)

Reply:
queue-proxy was constantly failing to establish connectivity with autoscaller, had to fix it to be able to prove that domain name change it working.